templates/project/.gitignore.twig

```diff
@@ -13,6 +13,9 @@ phpstan.neon
 {% if project.hasDocumentation %}
 /docs/_build/
 {% endif %}
+{% if project.usesPanther %}
+/tests/App/public/bundles
```

Comment:
This is related to https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/blob/3.x/tests/bootstrap.php#L52-L53
```
$publicDir = __DIR__.'/App/public';
$_SERVER['PANTHER_WEB_SERVER_DIR'] = $publicDir;
```

So I would recommend to add these two lines in the boostrap.php file (and not the custom one) if the panther option is set to true.

Reply:
I added the variable as a server variable in phpunit.xml.dist